File: app/src/main/java/com/nimtego/itunes/presentation/information_view/model/AlbumDetailsModel.java
package com.nimtego.itunes.presentation.information_view.model;

import com.nimtego.itunes.presentation.main.model.SongModel;

import java.util.List;

import lombok.Builder;
import lombok.Data;
import lombok.ToString;

@Data
@Builder
@ToString
public class AlbumDetailsModel {
    private String albumName;
    private String albumArtistName;
    private String albumArtwork;
    private int albumId;
    private String albumArtWorkUrl;
    private List<SongModel> songs;
    private Double collectionPrice;
    private String releaseDate;
    private String wikiInformation;
}

File: app/src/main/java/com/nimtego/plectrum/presentation/old/information_view/album/model/AlbumDetailsModel.kt
package com.nimtego.plectrum.presentation.old.information_view.album.model

import com.nimtego.plectrum.presentation.old.main.model.SongModel

data class AlbumDetailsModel (
        val albumName: String? = null,
        val albumArtistName: String? = null,
        val albumArtwork: String? = null,
        val albumId: Int = 0,
        val albumArtWorkUrl: String? = null,
        val songs: List<SongModel>? = null,
        val collectionPrice: Double? = null,
        val releaseDate: String? = null,
        val wikiInformation: String? = null)